project/urls.py

```diff
@@ -31,7 +31,7 @@
 urlpatterns = [
     url(r"^favicon\.ico$", favicon_view),
     url(r"^admin/", admin.site.urls),
-    url(r"^api/", include(api_urls)),
+    url(r"^api/", include((api_urls, 'api'))),
```

Comment:
s/'/"/g

You should configure your editor to run `black` every time you save. That way you don't have to remember :)

Reply:
I just set up pre-commit to do this, isort, & check for merge conflicts. If you don't want that as part of a forced workflow I can keep those running only locally :)